README.md

```diff
@@ -29,7 +31,7 @@ Your answer...
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+the router gets request and sends it to the specific controller.
 ```
```

Comment:
:+1:  Specifically, the router defines which URLs trigger which controller actions.